docs/bgp-policy.md

```diff
@@ -232,3 +278,5 @@ Please refer to the corresponding [antctl page](antctl.md#bgp-commands).
   with Windows Nodes.
 - Advanced BGP features such as BGP communities, route filtering, route reflection, confederations, and other BGP policy
   mechanisms defined in BGP RFCs are not supported.
+
+[Advertise Pod IPs through BGP Confederation]: #advertise-pod-ips-through-bgp-confederation
```

Comment:
why not use an inline link, like in the ToC?
we rarely use this style in the documentation (unless we have a lot of repeated links).

Reply:
Done.